packages/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -125,6 +125,7 @@ private void prepareMediaRecorder(String outputFilePath) throws IOException {
     // There's a specific order that mediaRecorder expects. Do not change the order
     // of these function calls.
     if (enableAudio) mediaRecorder.setAudioSource(MediaRecorder.AudioSource.MIC);
+    if (enableAudio) mediaRecorder.setAudioEncodingBitRate(recordingProfile.audioBitRate);
```

Comment:
This is it guys. @mklim @tvolkert @BesartLaci @kev2513 

Do you not trust that this fixes it? Without this change, I couldn't have launched our app (which is in production with hundreds of users on both iOS and Android). Without this change the audio is garbled.

How do we get this merged?

Reply:
@cfchris some of our plugin PRs are in a state where we're transitioning ownership of them from the "core Flutter team" to community owners, and unfortunately, your PR got caught in that transition.  I apologize.  We'll try to make sure this gets unblocked in the next few days.

If you don't have resolution by next Friday (Oct 9), please ping me, and I'll make sure this is resolved.

Reply:
Thank you so much @tvolkert !

I know there are a bunch of people that are going to be very happy to have this fixed and switch back to the main repo instead of our various forks.